.golangci.yml

```diff
@@ -41,6 +43,7 @@ linters:
     - maligned
     - misspell
     - nakedret
+    - scopelint
```

Comment:
This is deprecated in favor of exportloopref, which we already have enabled.

https://github.com/kyoh86/scopelint#obsoleted